readme.txt

```diff
@@ -0,0 +1 @@
+GIT ASSIGNMENT
```

Comment:
change this message to "git assignment" and amend it to previous commit. do not create separate commit for this change.

Reply:
Required changes done.